yandex/resource_yandex_mdb_postgresql_cluster_test.go

```diff
@@ -311,6 +311,9 @@ func testAccCheckMDBPGClusterHasUsers(r string, perms map[string][]string) resou
 var defaultUserSettings = map[string]interface{}{
 	"conn_limit": int64(50),
 }
+var testAccMDBPGClusterConfigUpdatedCheckConnLimitMap = map[string]int64{
+	"alice": 42,
+}
 
 func testAccCheckUnmodifiedUserSettings(r string) resource.TestCheckFunc {
```

Comment:
Does conn_limit is different from other user's settings somehow? Now it's definitely not "UnmodifiedUserSetting" anymore.

Reply:
If we want to test it in a different way from other settings just rename it, if it's similar probably we have to delete the test.

Reply:
test function rename testAccCheckUnmodifiedUserSettings -> testAccCheckConnLimitUpdateUserSettings